recipes/nb_conda/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+fontconfig
```

Comment:
Yes, it is a test-time dependency for phantomjs.

At some point we need to circle back around for a phantomjs recipe:
https://github.com/conda-forge/staged-recipes/pull/692

...but in the meantime, this gets the job done.